src/amplitude-client.js

```diff
@@ -108,6 +108,10 @@ AmplitudeClient.prototype.init = function init(apiKey, opt_userId, opt_config, o
       }
     }
 
+    if (type(this.options.logLevel) === 'string') {
```

Comment:
If you define a default logLevel in https://github.com/amplitude/Amplitude-JavaScript/blob/master/src/options.js, the SDK will do that parsing and validation for you via: https://github.com/amplitude/Amplitude-JavaScript/blob/master/src/amplitude-client.js#L159-L188